### DISCUSSION TITLE
[Request] Add Ubooquity

### DISCUSSION THREAD

**Discussion by alexbt117** [tteck/Proxmox#521] [Ideas]
Would it be possible to add a script for creating a Ubooquity LXC in proxmox? I think that having the books on a separate bind mount to the LXC will be a good idea as it is decoupled from the app and easier to manage and backup. 
It is a java application with a web gui. 
https://vaemendis.net/ubooquity/

Thank you for your good work!

**Comment by tteck** [comment]
Not Open Source
> Redistribution of the Ubooquity software is not allowed without explicit authorization from its author (users must download it from the Ubooquity website: http://vaemendis.net/ubooquity).
